hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -614,6 +614,14 @@ public class HoodieWriteConfig extends HoodieConfig {
       .withDocumentation("Whether to enable commit conflict checking or not during early "
           + "conflict detection.");
 
+  public static final ConfigProperty<Boolean> WRITE_BULK_INSERT_UTC_TIMEZONE = ConfigProperty
+      .key("hoodie.bulk_insert.write.utc-timezone")
+      .defaultValue(false)
+      .sinceVersion("0.14.0")
+      .withDocumentation("Whether to use UTC timezone to the conversion between epoch for bulk insert tasks.\n"
+          + "By default false use local timezone  ");
+
```

Comment:
We shoud add this in `HoodieWriteConfig`, this is a Flink specific config option.